tailwind.config.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
         'mobile-card': '92%',
       },
       height: {
+        280: '17.5rem',
```

Comment:
Can we use `h-70`?
If `h-4` is `1rem`, then `17.5 * 4 = 70`rem

Reply:
Renamed it. Thx :)